src/main/resources/db/migration/V005__Process_events.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE process_events (
+  id BIGSERIAL PRIMARY KEY,
+  container VARCHAR(50) NOT NULL,
+  zipFileName VARCHAR(255) NOT NULL,
+  createdAt TIMESTAMP NOT NULL,
+  envelope_id VARCHAR(50) NULL,
+  event VARCHAR(100) NOT NULL,
+  reason TEXT NULL
+);
+
+CREATE INDEX process_events_container_zip_idx ON process_events (container, zipFileName);
+CREATE INDEX process_events_container_event_idx ON process_events (container, event);
+CREATE INDEX process_events_zip_idx ON process_events (zipFileName);
+
+INSERT INTO process_events (container, zipFileName, createdAt, envelope_id, event, reason) (
+  SELECT container, zipFileName, createdAt, envelope_id, status, reason
+  FROM envelope_state
+);
+
+ALTER TABLE envelopes
+  ADD COLUMN lastEvent VARCHAR(100) NOT NULL DEFAULT 'ENVELOPE_CREATED';
```

Comment:
When we are creating an envelope we are creating it with status ENVELOPE_CREATED but we are not putting any entries in process_events table?